src/pykeen/nn/representation.py

```diff
@@ -1022,8 +1021,10 @@ class WikidataTextRepresentation(LabelBasedTransformerRepresentation):
         from pykeen.pipeline import pipeline
 
         dataset = get_dataset(dataset="codexsmall")
-        entity_representations = WikidataTextRepresentation.from_dataset(dataset=dataset)
-
+        entity_representations = WikidataTextRepresentation.from_dataset(
+            dataset=dataset,
+            encoder="transformer",
+        )
```

Comment:
it is something about this indention. There does not seem to be trailing whitespace though 🤔